pom.xml

```diff
@@ -1633,8 +1633,8 @@
     <jamon-runtime.version>2.4.1</jamon-runtime.version>
     <jettison.version>1.3.8</jettison.version>
     <!--Make sure these joni/jcodings are compatible with the versions used by jruby-->
-    <joni.version>2.1.11</joni.version>
-    <jcodings.version>1.0.18</jcodings.version>
+    <joni.version>2.1.40</joni.version>
```

Comment:
are these the versions that match what's in jruby-complete for 9.2.13.0?

Reply:
No, that's 2.1.31. We should change to that?

Reply:
Changed it.

Reply:
Only certain versions of Joni and JCodings work together, so when they don't match what's in the jruby-complete jar you can get weird hard to diagnose runtime failures depending on the classpath ordering.

I have in my backlog filing a jira about making sure they match, so I think you'd be fine to skip it here if you want. (like if something goes wrong with the test run after changing it.)